packages/core/integration-tests/test/javascript.js

```diff
@@ -5308,8 +5309,10 @@ describe('javascript', function () {
         name: 'BuildError',
         diagnostics: [
           {
-            message: `Failed to resolve '@swc/helpers' from '${normalizePath(
-              require.resolve('@parcel/transformer-js/src/JSTransformer.js'),
+            message: `Failed to resolve '@swc/helpers' from '${escapeMarkdown(
```

Comment:
I added `escapeMarkdown` because my working directory contains an underscore. The test is failed if the path is not escaped.